api/RTCPeerConnectionIceErrorEvent.json

```
@@ -32,7 +32,7 @@
             "version_added": false
```

```suggestion
            "version_added": "14.1"
```

Confirmed with https://mdn-bcd-collector.appspot.com/tests/api/RTCPeerConnectionIceErrorEvent, and this was also part of https://trac.webkit.org/changeset/270101/webkit.